app/forms/work_form.rb

```diff
@@ -31,6 +31,11 @@ def persisted?
 
   attribute :version, :integer, default: 1
 
+  attribute :works_contact_email, :string
+  validates :works_contact_email, format: {
```

Comment:
Could this ever be invalid?

Reply:
With the current design, I don't think so? Unless someone patches in bad data on the Cocina side? Is that possible?

Reply:
Then the validation should be in cocina. If the validation failed, then the user couldn't fix it. Also, I think the form would fail validation but the user wouldn't be told why it failed, so I really don't think we want this.